### ISSUE TITLE
Live results

### ISSUE THREAD

**Issue by ukanuk** [matterpoll/matterpoll#436]
#### Summary

Many online strawpoll sites show the votes live as they're coming in

#### Additional context

Currently you have to End the poll to see results. I would prefer leaving the poll open for awhile and checking results in a day or two.

**Comment by kaakaa** [comment]
The feature for viewing live results has implemented as #431 (, but it has not been released yet). Is this what you want?

**Comment by ukanuk** [comment]
Yeah looks like you already have the right idea.

**Comment by kaakaa** [comment]
Thanks for confirming @ukanuk.
I'll close this issue, and please watch #431 and [v1.6.0 Milestone](https://github.com/matterpoll/matterpoll/milestone/16) for this feature. Feel free to reopen this if you come up with additional requirements.

**Comment by FFdhorkin** [comment]
Not sure whether to comment here or on #431, but given you mentioned reopening this...

Right now the live status on a multi-vote poll shows the number of votes, not the number of voters.

For example, I opened a poll that has 4 options and it says there are 29 votes. Max votes are set to 3. That could be 10 people voting or 29 and it's impossible to tell without ending the poll. (If I had created with `--progress`, I could peek, but I overlooked that and it doesn't look like it's possible to change a poll's settings once it has been opened.
